frontend/src/components/debiai/project/Models.vue

```diff
@@ -152,6 +156,7 @@ export default {
       searchBar: "",
       selectedModelIds: [],
       commonModelResults: true,
+      metadata: null,
```

Comment:
Is this used by something ?

Reply:
Thank you for your comment, it has been removed.